proto/rill/admin/v1/api.proto

```diff
@@ -211,6 +211,83 @@ service AdminService {
     };
   }
 
+  // CreateUsergroup creates a user group in the organization
+  rpc CreateUsergroup(CreateUsergroupRequest) returns (CreateUsergroupResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/usergroups",
+      body: "*"
+    };
+  }
+
+  // GetUsergroups returns the user group details
+  rpc GetUsergroup(GetUsergroupRequest) returns (GetUsergroupResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{organization}/usergroups/{usergroup}"};
+  }
+
+  // RenameUsergroup renames the user group
+  rpc RenameUsergroup(RenameUsergroupRequest) returns (RenameUsergroupResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/usergroups/{usergroup}",
+      body: "*"
+    };
+  }
+
+  // ListUsergroups lists the org's user groups
+  rpc ListUsergroups(ListUsergroupsRequest) returns (ListUsergroupsResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{organization}/usergroups"};
+  }
+
+  // RemoveUsergroup removes the user group from the organization
+  rpc RemoveUsergroup(RemoveUsergroupRequest) returns (RemoveUsergroupResponse) {
+    option (google.api.http) = {delete: "/v1/organizations/{organization}/usergroups/{usergroup}"};
+  }
+
+  // SetOrganizationUsergroupRole sets the role for the user group
+  rpc SetOrganizationUsergroupRole(SetOrganizationUsergroupRoleRequest) returns (SetOrganizationUsergroupRoleResponse) {
+    option (google.api.http) = {
+      put: "/v1/organizations/{organization}/usergroups/{usergroup}/role",
+      body: "*"
+    };
+  }
+
+  // RevokeOrganizationUsergroupRole revokes the organization-level role for the user group
+  rpc RevokeOrganizationUsergroupRole(RevokeOrganizationUsergroupRoleRequest) returns (RevokeOrganizationUsergroupRoleResponse) {
+    option (google.api.http) = {delete: "/v1/organizations/{organization}/usergroups/{usergroup}/role"};
+  }
+
+  // SetProjectUsergroupRole sets the role for the user group
+  rpc SetProjectUsergroupRole(SetProjectUsergroupRoleRequest) returns (SetProjectUsergroupRoleResponse) {
+    option (google.api.http) = {
+      put: "/v1/organizations/{organization}/projects/{project}/usergroups/{usergroup}/roles",
+      body: "*"
+    };
+  }
+
+  // RevokeProjectUsergroupRole revokes the project-level role for the user group
+  rpc RevokeProjectUsergroupRole(RevokeProjectUsergroupRoleRequest) returns (RevokeProjectUsergroupRoleResponse) {
+    option (google.api.http) = {
+      delete: "/v1/organizations/{organization}/projects/{project}/usergroups/{usergroup}/roles",
+    };
+  }
```

Comment:
It would be nice to align these names with the ones for managing org and project member users, like `AddProjectMember`, `SetProjectMemberRole`, etc. As discussed previously, we might rename `Member` to `MemberUser` in those cases. Then we might get:

- `ListOrganizationMembers` -> `ListOrganizationMemberUsers` 
- `AddOrganizationMember` -> `AddOrganizationMemberUser`
- `RemoveOrganizationMember` -> `RemoveOrganizationMemberUser`
- `SetOrganizationMemberRole` -> `SetOrganizationMemberUserRole`
- `ListProjectMembers` -> `ListProjectMemberUsers`
- `AddProjectMember` -> `AddProjectMemberUser`
- `RemoveProjectMember` -> `RemoveProjectMemberUser`
- `SetProjectMemberRole` -> `SetProjectMemberUserRole`
- `ListOrganizationMemberUsergroups` 
- `AddOrganizationMemberUsergroup`
- `RemoveOrganizationMemberUsergroup`
- `SetOrganizationMemberUsergroupRole`
- `ListProjectMemberUsergroups`
- `AddProjectMemberUsergroup`
- `RemoveProjectMemberUsergroup`
- `SetProjectMemberUsergroupRole`